application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -81,7 +78,16 @@ private boolean wasThreadAlreadyHandled(long threadChannelId) {
     }
 
     private void handleHelpThreadCreated(ThreadChannel threadChannel) {
-        helper.writeHelpThreadToDatabase(threadChannel.getOwnerIdLong(), threadChannel);
+        threadChannel.retrieveMessageById(threadChannel.getIdLong()).queue(message -> {
+
+            long authorId = threadChannel.getOwnerIdLong();
+
+            if (isPostedByBot(message)) {
+                authorId = message.getMentions().getUsers().get(0).getIdLong();
```

Comment:
not obvious what this code does if ur not familiar with the feature. needs a comment or a helper method instead